spec/requests/applications_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'swagger_helper'
+
+RSpec.describe 'applications', type: :request do
+
+  path '/api/v1/applications' do
+    
+    get 'edit an application' do
+      produces 'application/json'
+      parameter name: :id, in: path, type: string 
+
+        response '200' 'edit made' do
+          schema type: :object,
+            properties: {
```

Comment:
I'm psyched to see this working and hopefully actually generating swagger files. Looking forward to seeing this with actual application model attributes.